modules/gitops-verifying-argo-cd-self-healing-behavior.adoc

```diff
@@ -0,0 +1,34 @@
+[id="verifying-argo-cd-self-healing-behavior_{context}"]
+= Verifying Argo CD self-healing behavior
+
+Argo CD constantly monitors the state of the deployed applications, detects differences between the specified manifests in Git and live changes in cluster, and automatically corrects them. This behavior is referred to as self-healing. 
+
+You can learn more about self-healing in link:https://argo-cd.readthedocs.io/en/stable/user-guide/auto_sync/#automatic-self-healing[Argo CD documentation].
+
+This section describes the steps to test and observe the self-healing behavior.
+
+.Prerequisites
+
+* The application was deployed and configured.
+
+.Procedure
+
+. In the Argo CD dashboard, verify that application has the `Synced` status.
+
+. Click the `app-spring-petclinic` tile in the Argo CD dashboard to view the application resources that are deployed to the cluster.
+
+. In the OpenShift Web Console, navigate to the Developer perspective.
+
+. Modify the Spring PetClinic deployment and commit the changes to the `app/` directory of the Git repository. Argo CD will automatically deploy the changes to the cluster.
+
+. Test the self-healing behavior by modifying the deployment on the cluster and scaling it up to two pods while watching the application in the OpenShift Web Console:
```

Comment:
When I read an instruction followed by a colon I expect to see the step I need to do. If the next sentence also ends in a colon in my opinion that is confusing. Maybe the first sentence could end in a full stop?